walkthroughs/week-3-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java

```diff
@@ -37,7 +37,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     // Convert the server stats to JSON
     ServerStats serverStats = new ServerStats(startTime, currentTime, maxMemory, usedMemory);
-    String json = convertToJson(serverStats);
+    String json = convertToJsonUsingGson(serverStats);
```

Comment:
Should you be submitting changes to example files? My intuition is no but I might be wrong.

Reply:
Ok, will do!

Reply:
Is this one and the one from the last PR the only two changes?